Sofa/Component/Collision/Detection/Algorithm/src/sofa/component/collision/detection/algorithm/DirectSAPNarrowPhase.cpp

```diff
@@ -31,8 +31,7 @@ namespace sofa::component::collision::detection::algorithm
 using namespace sofa::component::collision::geometry;
 
 DirectSAPNarrowPhase::DirectSAPNarrowPhase()
-        : d_draw(initData(&d_draw, false, "draw", "enable/disable display of results"))
```

Comment:
Welcome to the depreciation world 😈
You remove a data, so ideally it would need depreciation + warning.🤐 
You will break the compilation for somebody this data somwehere + you are changing a "runtime" behavior so the final user should at least be aware of that with messages somewhere

Reply:
Hah it crossed my mind after I made the pull request. I kind of hoped that it would go unnoticed...